__tests__/app.tests.js

```diff
@@ -0,0 +1,37 @@
+const app = require('../app');
+const request = require('supertest');
+const data = require('../db/data/test-data');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection.js');
+
+afterAll(()=>{
+    return db.end();
+})
+beforeEach(()=>{
+    return seed(data)
+})
+
+describe('app',()=>{
```

Comment:
I'd probably recommend you seperate these describe blocks out, so this out be the describe block for `/api/topics` instead of `app`